src/pack/blob.ts

```diff
@@ -1,12 +1,14 @@
 import { Blob } from "@web-std/blob"
 import all from 'it-all'
-import { ImportCandidateStream } from 'ipfs-core-types/src/utils'
+import { ImportCandidateStream as ImportCandidateStreamT } from 'ipfs-core-types/src/utils'
 
 import { Blockstore } from '../blockstore'
 import { MemoryBlockStore } from '../blockstore/memory'
 
 import { pack } from './index'
 
+export type ImportCandidateStream = ImportCandidateStreamT
```

Comment:
I think we could do something like:
```suggestion
export type { ImportCandidateStream }
```

see: https://github.com/ipfs-shipyard/nft.storage/blob/bc0201cb7f8d2b4a6114f3071203c4e685a14066/packages/client/src/lib/interface.ts#L1-L2

Reply:
oh I see, that is because of the `import type`. Without it TS complains

Good catch